src/components/loading/loading.js

```diff
@@ -89,6 +90,9 @@ class Loading extends BaseComponent {
     }
 
     this.player.addEventListener(this.player.Event.PLAYER_STATE_CHANGED, e => {
+      if (!this.state.afterPlayingEvent) {
```

Comment:
I think we are starting to have duplicate logic here, and maybe it's better to consider moving to playback events based state only and not state change event.
You already see that we both register to playing event a few line below and do the same action there and also in here in state change to playing.